angular.json

```diff
@@ -89,6 +89,11 @@
                 "input": "src/assets",
                 "output": "/assets"
               },
+              {
+                "glob": "**/*",
+                "input": "src/assets/documents",
+                "output": "/"
+              },
```

Comment:
don't need this, as it's already in master

Reply:
this has now been removed.